dependencies.gradle

```diff
@@ -0,0 +1,5 @@
+// Add your dependencies here
+
+dependencies {
```

Comment:
Add Better questing as GTNH dependency. It is already migrated. There are two versions of apis for this mod: `api` and `api2`. I am not sure which one you need to add, maybe both. Try try to add it as compileOnly first and see if it runs.

Reply:
And you must remove the libs folder for that.

Reply:
what is strange is nitpick can't find api2:

```
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find BetterQuesting-master-SNAPSHOT-api2.jar (com.github.GTNewHorizons:BetterQuesting:master-SNAPSHOT:3.0.350-GTNH-g81d7f81-10).
     Searched in the following locations:
         https://jitpack.io/com/github/GTNewHorizons/BetterQuesting/master-SNAPSHOT/BetterQuesting-master-3.0.350-GTNH-g81d7f81-10-api2.jar
```

but :dev works fine

Reply:
(and api doesn't work)

Reply:
Need my pc for that. Might have time for it tomorrow